code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -636,6 +636,24 @@
 	untrackable = TRUE
 	access = list(ACCESS_SYNDICATE, ACCESS_SYNDICATE_LEADER, ACCESS_SYNDICATE_COMMAND, ACCESS_EXTERNAL_AIRLOCKS)
 
+// like /obj/item/card/id/syndicate, but you can only swipe access, not change your identity, its also trackable
+/obj/item/card/id/syndi_scan_only
+	name = "Syndicate Operative's ID card (Operative)"
+	rank = "Operative"
+	assignment = "Operative"
+	registered_name = "Syndicate Operative"
+	access = list(ACCESS_SYNDICATE)
```

Comment:
maybe add something to examine to show traitors that this is different than an agent ID card?